app/src/main/java/com/example/rmservicelocator/room/MyDatabaseApplication.kt

```diff
@@ -0,0 +1,28 @@
+package com.example.rmservicelocator.room
+
+import android.app.Application
+import android.content.Context
+import androidx.room.Room
+import com.example.rmservicelocator.service.ServiceLocator.init
+
+class MyDatabaseApplication : Application() {
+
+    var appDatabase: AppDatabase? = null
+
+    override fun onCreate() {
+        super.onCreate()
+
+        init(this)
```

Comment:
лучше писать так, а то не понятно у какого класса вызывается init
```suggestion
        ServiceLocator.init(this)
```